contracts/Raphael.sol

```diff
@@ -42,14 +42,11 @@ contract Raphael is ERC721Holder, Ownable, ReentrancyGuard {
 
     bool private shutdown = false;
 
-    uint256 public CREATE_TO_VOTE_PROPOSAL_DELAY = 18500; // ~3 days
-    uint256 public VOTING_DURATION = 30850; // ~5 days
+    uint256 public CREATE_TO_VOTE_PROPOSAL_DELAY = 13091; // ~2 days
+    uint256 public VOTING_DURATION = 91636; // ~14 days
 
-    // comment out for testing
-    // uint256 public constant MIN_DURATION = 18500;
-    // remove testing value for production
-    uint256 public constant MIN_DURATION = 5; // testing value
-    uint256 public constant MAX_DURATION = 190000; // ~1 month
+    uint256 public constant MIN_DURATION = 5; // ~ 1 minute
```

Comment:
I maintain that the minimum should be longer, but if this has been approved by the community, they're the final word.

Reply:
resolving conversation after being informed the community decision is to keep the minimum threshold low